tests/test_0852-fix-strided-interp-extra-offsets.py

```diff
@@ -4,8 +4,6 @@
 import uproot
 import skhep_testdata
 
-ROOT = pytest.importorskip("ROOT")
```

Comment:
Right, because the file already exists. This will allow the test to be tested on more systems. Thanks for catching this!